documentation/modules/ROOT/pages/connectors/postgresql.adoc

```diff
@@ -1816,8 +1816,9 @@ It is possible to capture changes in a PostgreSQL database that is running in li
 * Initiate logical replication from an AWS account that has the `rds_replication` role.
   The role grants permissions to manage logical slots and to stream data using logical slots.
   By default, only the master user account on AWS has the `rds_replication` role on Amazon RDS.
-  To initiate logical replication from a user account other than the master account, grant the account the `rds_replication` role.
-  For example, `grant rds_replication to _<my_user>_`
+  To enable a user account other than the master account to initiate logical replication, you must grant the account the `rds_replication` role.
+  For example, `grant rds_replication to _<my_user>_`. Superuser access is required to grant the role.
+  To enable the non-master account to perform an initial snapshot, the account also requires `SELECT` permission on the schema of any tables to be captured.
```

Comment:
> on the schema of any tables

This is interesting; so SELECT permissions are scoped per schema, not per table? If it's per table actually, the part "the schema of" should be removed.

Reply:
Perhaps I misunderstood your comment in the Jira (or took too many liberties in my rewording):

_the user also need to have SELECT grants for the schema(s) hosting the tables which should be captured, in order to do the initial snapshot._

Reply:
Ah, I see. Yeah, so that'd be something to confirm with upstream Postgres docs at which level those GRANTs apply; I'd actually expect table level. Can you try and figure it out?

Reply:
The[ Logical replication security section in the PG docs](https://www.postgresql.org/docs/10/logical-replication-security.html) mentions a requirement for `SELECT` permissions on the table to be replicated, but says nothing about schema permissions. It also mentions several other requirements. I should probably direct users to read the PG docs for details, eh?

Reply:
The doc that I linked to is for PGv10. If I do link the PG doc, should I target a specific version? Do we need a `postgres-version` attribute?